### PR TITLE
feat(deploy to netlify): deployement of the project to Netlify

### LINKED ISSUE #36: Deploy to Netlify
**Task title**
Deploy to Netlify

**Task description**
Deploy to Netlify so we have access to more collaborative tools on PR's previews.

### PR DESCRIPTION
**Pull request by M07am3dM**
Deploy to Netlify so we have access to more collaborative tools on PR's previews.

fix #36

# Related Issue

- Resolve #36

### CODE REVIEW

**Comment by dzc0d3r** [comment]
<details><summary>Comment has been resolved</summary>


Testing recording! 💯 
@M07am3dM 
recording


<details><summary>Browser metadata</summary>





```
Path:      /ar
Browser:   Firefox 115.0 on Linux x86_64
Viewport:  1366 x 615 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-37--innerspace.netlify.app%2Far&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=115.0&resolution=1366x615&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-37--innerspace.netlify.app/ar) · [Mark as Unresolved](https://app.netlify.com/cdp/resolve?deployID=652bcfb68a4f36000841d7d5&commentID=652be39a14031e364c8705e5&resolution=unresolved)

</details>

**Comment by dzc0d3r** [comment]
<details><summary>Comment has been resolved by Walid Lamraoui</summary>

what font we used ?
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /ar
Browser:   Firefox 115.0 on Linux x86_64
Viewport:  1366 x 615 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-37--innerspace.netlify.app%2Far&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=115.0&resolution=1366x615&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-37--innerspace.netlify.app/ar) · [Mark as Unresolved](https://app.netlify.com/cdp/resolve?deployID=652bcfb68a4f36000841d7d5&commentID=652be4b61393ec6c536b2c7b&resolution=unresolved)

</details>

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 -   Project name: Online Therapist
 -   Project description: This project will allow users to schedule an appointment with a therapist and take therapy sessions online through chat, video, and voice calls. Users can search for information on the website, and also there will be tests or therapist activities for users.
--   Technology used: Nextjs, TailwindCSS, i18next, Husky, Prettier, Eslint, Commitlinter (Conventional Commits)
+-   WebSite link: https://innerspace.netlify.app
```
Comment (dzc0d3r) [inline README.md:7]: Great :100:  
Thank you @M07am3dM :pray:

**Review by dzc0d3r — APPROVED** [review]
Looks Great :tada: 
Thank you @M07am3dM

**Review by Lilicod — APPROVED** [review]
Looks great 💯 
Great job @M07am3dM

**Comment by M07am3dM** [comment]
well done

<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 118.0.0.0 on Windows 10
Viewport:  1536 x 715 @1.25x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-37--innerspace.netlify.app%2F&os=Windows&os_version=10&browser=Chrome&browser_version=118.0&resolution=1536x715&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-37--innerspace.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=652bcfb68a4f36000841d7d5&commentID=652c505cac53123a29fa9fa7&resolution=resolved)

**Pull request merged**